CHANGELOG.md

```diff
@@ -12,7 +12,8 @@
 - [](https://github.com/vegaprotocol/vega/issues/xxxx) -
 
 ### 🐛 Fixes
-- [](https://github.com/vegaprotocol/vega/issues/xxxx) -
```

Comment:
This probably should stay as template for the next guy?

Reply:
I think it's fine, the next guy will just grab one of the existing ones, that's how it usually goes.